index.html

```diff
@@ -182,6 +182,28 @@ <h1>The Registry</h1>
   "owner": "did:example:123456789abcdefghi",
   "publicKeyPem": "-----BEGIN PUBLIC KEY...END PUBLIC KEY-----\r\n"
 }
+</pre>
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+          EdDsaSAPublicKeySecp256k1
```

Comment:
This spec is super old... it needs some love. Pulling the PR because not including the key/suite is an omission.